### PR TITLE
Export passed schemes for fast pass, edip, sdip candidates

### PR DESCRIPTION
**Pull request by satishkamatam**

### CODE REVIEW

**Comment by henricook** [comment]
LGTM at a skim, i understand this is @benjstephenson 's baby

File: test/services/application/ApplicationServiceSpec.scala [hunk 1]
```diff
@@ -76,6 +80,85 @@ class ApplicationServiceSpec extends UnitSpec with ExtendedTimeout {
...
+
+    "retrieve passed schemes for Faststream application with fast pass approved" in new ApplicationServiceTest {
+
+      val userId = "userId"
+      val applicationId = "appId"
+      val frameworkId = ""
+
+      val faststreamApplication = ApplicationResponse(applicationId, "", ApplicationRoute.Faststream,
+        userId,ProgressResponse(applicationId, fastPassAccepted = true), None)
+      val passmarkEvaluation = PassmarkEvaluation("", None,
```
Comment (benjstephenson) [inline test/services/application/ApplicationServiceSpec.scala]: Do we need this val here?

**Comment by benjstephenson** [comment]
👍  LGTM!

File: test/services/application/ApplicationServiceSpec.scala [hunk 2]
```diff
@@ -76,6 +80,82 @@ class ApplicationServiceSpec extends UnitSpec with ExtendedTimeout {
...
+      when(evalPhase1ResultMock.getPassmarkEvaluation(eqTo(applicationId))).thenReturn(Future.successful(passmarkEvaluation))
+
+      val passedSchemes = underTest.getPassedSchemes(userId, frameworkId).futureValue
+
+      passedSchemes mustBe List(SchemeType.Sdip)
+
+    }
   }
 
   trait ApplicationServiceTest {
```
Comment (benjstephenson) [inline test/services/application/ApplicationServiceSpec.scala]: A bit pedantic, but there are no negative tests here to check that the failed schemes are filtered out.

**Pull request merged**